doc/plugin_agent_workloadattestor_k8s.md

```diff
@@ -7,8 +7,10 @@ the kubelet for information about the pod.
 | Configuration | Description |
 | ------------- | ----------- |
 | kubelet_read_only_port | The port on which the kubelet has exposed its read-only API. |
+| max_poll_attempts | number of polling attempts against the kublet to find pod information |
+| poll_retry_interval | how long the plugin waits between polling attempts |
```

Comment:
These knobs are useful for sufficiently advanced users, but I think that they may be confusing for a lot of people. I always try to lean towards less knobs when possible, as it (in the general case) makes user experience better, especially for folks new to the project and in cases that can have adverse effect on performance or security. Perhaps we can just leave these as undocumented configurables?

Reply:
Sounds good.